demos/isoline_simplification/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(SOURCES
+    isoline_simplification_demo.cpp
+)
+
+add_executable(isoline_simplification_demo ${SOURCES})
+
+target_link_libraries(
+    isoline_simplification_demo
+    PRIVATE
+    ${COMMON_CLA_TARGET}
+    core
+    renderer
+    isoline_simplification
+    CGAL::CGAL
+    Qt5::Widgets
+    glog::glog
```

Comment:
Same as above, I don't think glog is needed here.

Reply:
Fixed